### ISSUE TITLE
[Bug]:

### ISSUE THREAD

**Issue by Lady-Fitzgerald** [FreeTubeApp/FreeTube#7072]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

 The [BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed has reared it's ugly head again. Just now—12:33PM MST, I keep get the message despite multiple attempts. Right now, FreeTube is essentially useless.

### Expected Behavior

Open up videos.

### Issue Labels

accessibility issue, content not loading, feature stopped working

### FreeTube Version

v0.23.2 Beta

### Operating System Version

Linux Mint 21.1 Cinnamon x64

### Installation Method

.deb

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.23.2 Beta

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).
